src/components/Images/Images.js

```diff
@@ -2,13 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import useRequired from '../../hooks/useRequired.js'
 import useInvalid from '../../hooks/useInvalid.js'
-
 import { createCloudinaryLegacyURL } from '@cloudinary/url-gen'
-
-import { v4 as uuidv4 } from 'uuid'
-// eslint-disable-next-line
-import lazysizes from 'lazysizes'
```

Comment:
we no longer rely on `lazysizes` since now `<img>` tag natively supports `loading="lazy"`

Reply:
@ChirashiSushi Firefox only lists partial support and Safari says it requires an experimental feature to be enabled. I don't think this would qualify as native support.

Reply:
I left a comment here originally but misread the spec, I see it's only iframes that don't have full support. LGTM!